docs/database/readme.md

```diff
@@ -0,0 +1,159 @@
+# Schema
+
+```mermaid
+---
+title: Airneis database model
+---
+
+classDiagram
+
+Categories <|-- Products
```

Comment:
We would like to query also all products of categories if not mistaken when making filters for example. You could add the relation one-to-many from categories to products as one category may have many products.

Reply:
You can check the docuementation here : https://mermaid.js.org/syntax/classDiagram.html#cardinality-multiplicity-on-relations